packages/happo-target-react-native/scripts/build-runner-android.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+echo "Building HappoRunner.apk"
+node ./node_modules/react-native/local-cli/cli.js run-android --root ./runner
```

Comment:
I worry a bit about the hard-coded node_modules path in these scripts. Do these need to be shell scripts or can they be fully contained in package.json scripts?